controllers/healthcheck_test.go

```diff
@@ -0,0 +1,87 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/gorilla/mux"
+	"github.com/prest/prest/mocks"
+	"github.com/prest/prest/testutils"
+)
+
+func TestHealthStatus(t *testing.T) {
+	var testCases = []struct {
+		description string
+		url         string
+		method      string
+		status      int
+	}{
+		{"Healthcheck endpoint", "/_health", "GET", http.StatusOK},
+	}
+
+	router := mux.NewRouter()
+	dbConn := SDbConnection{}
+	router.HandleFunc("/_health", WrappedHealthCheck(dbConn)).Methods("GET")
+	server := httptest.NewServer(router)
+	defer server.Close()
+
+	for _, tc := range testCases {
+		t.Run(tc.description, func(t *testing.T) {
+			testutils.DoRequest(t, server.URL+tc.url, nil, tc.method, tc.status, "HealthStatus")
+		})
+	}
+}
+
+func TestMockedHealthcheckFailedConnection(t *testing.T) {
+	var testCases = []struct {
+		description string
+		url         string
+		method      string
+		status      int
+	}{
+		{"Healthcheck endpoint failed connection", "/_health", "GET", http.StatusServiceUnavailable},
+	}
```

Comment:
I think I have asked that before, but… if we have only one test case, why build a test case table? Are we adding more tests?

Reply:
There is no need for it, but, I chose to stand with it to keep the pattern observed on previous tests. I can simplify this for sure. Do you think it would be better, cuduquitos <@cuducos> ?